src/storeshield/dependencies/container.py

```
@@ -0,0 +1,113 @@
+import ipinfo
+import redis.asyncio as redis
+from apscheduler.jobstores.redis import RedisJobStore
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
```

Ruff (0.8.2)

`apscheduler.schedulers.asyncio.AsyncIOScheduler` imported but unused

Remove unused import: `apscheduler.schedulers.asyncio.AsyncIOScheduler`

(F401)